site/get-started.md

```diff
@@ -62,3 +62,12 @@ template listing available configuration keys and their default values.
 be restricted to either their default values, or the values set in the Spark configuration used by Livy.
 * **log4j.properties:** configuration for Livy logging. Defines log levels and where log messages will be written to.
 The default configuration template will print log messages to stderr.
+
+### 4. Start using Livy
+
+Once the Livy server is running, you can connect to it, by default, on port 8998 (this can be changed with the 
```

Comment:
nit: I think there's too many commas here. This can probably just be: `you can connect to it on port 8998 (the port can be`